packages/core/middleware/src/createServer.ts

```diff
@@ -42,8 +42,7 @@ function createServer (config: MiddlewareConfig): Express {
 
       res.send(platformResponse);
     } catch (error) {
-      res.status(500);
-
+      res.status(error.networkError.statusCode);
```

Comment:
```suggestion
      res.status(error.networkError.statusCode || 500);
```
Maybe like that?

Reply:
This will work only with Apollo GraphQL client. Other GraphQL or REST clients will throw errors with different structure.

Reply:
@Fifciu @filipsobol Guys, I've created a little helper with a purpose to agnosticize getting the status from various error objects provided by different integrations. Could this be a more versatile solution?

Reply:
@lsliwaradioluz quick reminder - `null` also has type `object`

Reply:
Thank you @Razz21